test/integration/pages/util/util.py

```diff
@@ -0,0 +1,30 @@
+"""Tool for creating class names."""
+import hashlib
+import os
+from functools import reduce
+
+table = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-_'
+
+class_names = {
+    'button': 'webextension/widgets/button.css',
+    'ghost-theme': 'webextension/widgets/button.css',
+    'homepage': 'webextension/manage/components/homepage.css',
+    'item-summary': 'webextension/manage/components/item-summary.css',
+    'minimal': 'webextension/widgets/button.css',
+    'title': 'webextension/manage/components/item-summary.css',
+    }
```

Comment:
Rather than mapping class names to filenames, why not create a mapping for some kind of unique name to the full munged class name? For example:

```python
class_names = {
    'button.minimal': munged_class_name('webextension/widgets/button.css', 'minimal'),
    # ...
}
```

Reply:
@jimporter would that be easier to maintain? This is really for you, so easy updates are priority I feel.

Reply:
I think having different names for the same value is not ideal. If something is a button persay, we should just have it be ```button```, not ```minimal``` or anything else, as far as accessing its css for the test.